CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.6]
+
+### Added
+
+- Checking for PagerDuty Admins
```

Comment:
Thanks for adding to the `CHANGELOG`! I really need to get in a habit of checking for these 😁

I don't think we must change this since it's tagged and ready to go, but I would recommend something like:

```suggestion
- Added `pagerduty_user.admin` property
```

Reply:
I'll fix it the next time I make a change, good input.